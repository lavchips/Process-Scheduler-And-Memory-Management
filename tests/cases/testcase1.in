./scheduler -f processes.txt -a ff -m u

/****************************/
./scheduler -f benchmark-cs.txt -a cs -s 100 -m v

better than

./scheduler -f benchmark-cs.txt -a rr -s 100 -m v -q 10
./scheduler -f benchmark-cs.txt -a ff -s 100 -m v

/******************************/
./scheduler -f benchmark-cm.txt -a rr -s 100 -m cm -q 10

better than

./scheduler -f benchmark-cm.txt -a rr -s 100 -m v -q 10
./scheduler -f benchmark-cm.txt -a rr -s 100 -m p -q 10


0 1 40 11
12 2 40 40
24 3 40 12
36 4 40 40
46 5 80 15
50 6 40 100
60 7 80 40
70 8 20 10
80 9 4 11
90 10 4 15
100 11 4 80
110 12 4 1