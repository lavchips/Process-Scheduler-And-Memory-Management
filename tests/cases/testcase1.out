0, RUNNING, id=4, remaining-time=30
30, FINISHED, id=4, proc-remaining=3
30, RUNNING, id=2, remaining-time=40
70, FINISHED, id=2, proc-remaining=2
70, RUNNING, id=1, remaining-time=20
90, FINISHED, id=1, proc-remaining=1
90, RUNNING, id=3, remaining-time=30
120, FINISHED, id=3, proc-remaining=0
Throughput 2, 1, 3
Turnaround time 71
Time overhead 4.25 2.56
Makespan 120
